tools/ovc/openvino/tools/ovc/moc_frontend/pipeline.py

```diff
@@ -90,156 +92,164 @@ def check_places_are_same(places_original: List[Place], places_new: List[Place])
             [item for item in places_original if any(
                 [item.is_equal(item2['node']) for item2 in places_new])]) == len(places_original)
 
-    def add_names_to_tensors(model: InputModel, places: List[Place]):
-        """
-        Adds additional names to some model input tensors. This helper should be used
-        when a model modification is going to happen.
-        :param model The input model loaded by a given frontend
-        :param places An object containing Places and names that will be used for model modification
-        """
-        for new_input in places:
-            if 'input_name' not in new_input:
-                continue
-            try:
-                model.add_name_for_tensor(new_input['node'], new_input['input_name'])
-            except NotImplementedFailure as e:
-                # some frontends might not implement this method
-                log.warning('Could not add an additional name to a tensor pointed to by \'{}\'. Details: {}'.format(
-                    new_input['input_name'], str(e)))
-
-    enabled_transforms, disabled_transforms = get_enabled_and_disabled_transforms()
-    if 'ANALYSIS_JSON_PRINT' in enabled_transforms:
-        # NOTE that model analysis is performed before applying user's settings (inputs's shapes etc.)
-        framework_model = moc_front_end.decode(input_model)
-        json_model_analysis_dump(framework_model)
-        # a model is not processed further in json analysis mode
-        sys.exit(0)
```

Comment:
why it is moved. please leave it here

Reply:
This code is also needed for pytorch, it is not affected by changes in args.input and other variables so it can be moved in the beginning